modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/email/provider/RoleEmailProvider.java

```diff
@@ -110,16 +158,17 @@ private String _getEmailAddresses(
 					NotificationRecipientSettingConstants.NAME_ROLE_NAME));
 
 			if ((role == null) ||
-				(role.getType() != RoleConstants.TYPE_ACCOUNT)) {
+				((role.getType() != RoleConstants.TYPE_ACCOUNT) &&
+				 (role.getType() != RoleConstants.TYPE_ORGANIZATION))) {
```

Comment:
Do we only want to `getEmailAdresses` if the role is both Type Account **and** Organization? Should this be an **or**?

Reply:
Currently, we only support Account and Organization roles. So, if the role type is not account **and** is not organization, I'm calling `continue`.

Reply:
I mixed them up. My bad! :sweat_smile: